app/assets/stylesheets/practices.scss

```diff
@@ -0,0 +1,54 @@
+.profile {
+  margin: none;
+  border-bottom: 1px solid #ddd;
+  padding: 10px;
+  background-color: #fff;
+  .name {
+    h3 {
+      margin: 10px;
+    }
+  }
+}
+
+.practices-list {
+  margin: 20px 5px;
+  list-style-type: none;
+  .practice {
+    background-color: #fff;
+    border: 1px solid #ddd;
+    border-radius: 3px;
+    .title {
+      margin: none;
+      border-bottom: 1px solid #ddd;
+      .fa.fa-check-square-o {
+        text-align: center;
+        float: left;
+        margin: 10px
+      }
+      h3 {
+        margin: 10px;
+        font-weight: 900;
+      }
+    }
+    .daily-progress {
+      padding: 10px;
+      border-bottom: 1px solid #ddd;
+    }
+    .total, .longest-streak {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line